src/OrchardCore/OrchardCore.DisplayManagement.Liquid/TemplateOptionsConfigurations.cs

```diff
@@ -55,13 +60,13 @@ public void Configure(TemplateOptions options)
                 return new ArrayValue(forms.Keys.Select(x => new StringValue(x)).ToArray());
             }
 
-            return new ArrayValue(forms[name].Select(x => new StringValue(x)).ToArray());
```

Comment:
Do I understand correctly that this is non-breaking?

Reply:
That’s the plan. The new `StringValuesValue` should behave similarly to the existing `ArrayValue` in Liquid, but with the added flexibility of being usable as a single string.

This would allow templates like:

```liquid
{% if Request.Query.Greeting == "hello" %}
Hello, world!
{% endif %}
```

or:

```liquid
{% if Request.Query.Greeting[0] == "hello" %}
Hello, world!
{% endif %}
```

Currently, the second example is the only correct way to access form values, query parameters, and headers. This change should be non-breaking, but it will simplify template authoring and improve readability going forward.